fdbserver/DataDistribution.actor.h

```diff
@@ -390,19 +390,21 @@ struct StorageWiggleMetrics {
 };
 
 struct StorageWiggler : ReferenceCounted<StorageWiggler> {
+	enum State : uint8_t { INVALID = 0, RUN, PAUSE };
```

Comment:
For states stored in the database, the best practice is to set explicit integer values for each entry so it's clear that the list cannot be arbitrarily modified in the future.